lektor/admin/package.json

```diff
@@ -36,6 +36,8 @@
     "react-router": "^2.6.0",
     "standard": ">8.5.0",
     "style-loader": "^0.8.3",
+    "tui-editor": "^1.3.3",
```

Comment:
1.4.0 has been released, maybe that should be used?

Reply:
Checked it out, and there are no issues with using 1.4.0 so this has been added.